<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>set_sequence_layout</class>
 <widget class="QDialog" name="set_sequence_layout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Files</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="sequence_group">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;With this option you reset the technical sequence id on the database. &lt;/p&gt;&lt;p&gt;Do it only when you know what you are about to do.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>Reset the technical sequence used as t_id value</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="sequence_label">
        <property name="text">
         <string>T_Id (Sequence) Value</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sequence_value_edit">
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="pg_use_super_login">
        <property name="text">
         <string>Use logins from settings (superuser) for this operation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
